packages/assets-controllers/src/token-prices-service/codefi-v2.ts

```diff
@@ -155,6 +147,12 @@ export const SUPPORTED_CURRENCIES = [
   'sats',
 ] as const;
 
+/**
+ * The zero address
```

Comment:
Remember that we can write whatever we want for a JSDoc block. Usually, we go into more detail about the thing that we are writing the JSDocs for, but in this case I think it is fairly obvious (both from the name and also the value) that this is a zero address. So instead of saying what it is, what are your thoughts on explaining _why_ it's needed? What purpose does this serve and _how_ is it used?